sailthru/requirements/base.txt

```diff
@@ -1,2 +1,3 @@
+nose==1.3.7
```

Comment:
@ahsan-ul-haq there's no need for a nested requirements file here. Please just use a requirements.txt.